src/header.rs

```diff
@@ -7,6 +7,9 @@ use std::time::Duration;
 /// Custom HTTP header for expiry dates.
 pub const EXPIRE: &str = "expire";
 
+/// Custom HTTP header to override filename.
+pub const FILENAME: &str = "filename";
```

Comment:
Can we make this constant private (or maybe pub crate) if we are not using it anywhere else? What do you think?

Reply:
yes, I think this makes sense. done.